satpy/readers/ahi_hsd.py

```diff
@@ -583,6 +612,15 @@ def convert_to_radiance(self, data):
         else:
             gain = self._header["block5"]["gain_count2rad_conversion"][0]
             offset = self._header["block5"]["offset_count2rad_conversion"][0]
+        # If using custom calibration from GSICS, apply it here
+        if self.calib_mode == "CUSTOM":
```

Comment:
Could you make a separate method for retrieving the gain and offset? It would make the code easier to follow.

Reply:
I think I've done this in my new commits, please check if that's what you wanted!